openshift/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This will produce and image to be used in OpenShift
+# Build should be triggered from the repo root like:
```

Comment:
What @csomh describes matches with the kind of policy that @puiterwijk and @nirik have been talking about for os.fedoraproject.org (I think) -- build in OSBS out of dist-git, keep no bc in openshift, but use a template to define a dc that pulls from the distro registry.